ramls/hridsettings.json

```diff
@@ -0,0 +1,38 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "The HRID settings",
+  "type": "object",
+  "javaType": "org.folio.rest.jaxrs.model.HridSettings",
+  "properties": {
+    "id": {
+      "description": "The ID of the one and only HRID settings record",
+      "type": "string",
+      "readonly": true,
+      "$ref": "uuid.json"
+    },
+    "instance": {
+      "description": "The HRID settings for instances",
+      "type": "object",
+      "$ref": "hridsetting.json",
+      "default": {"prefix": "in", "startNumber": 1}
```

Comment:
What does this property do? Does it set the values of those properties when they aren’t present in the record?

If so, what happens when settings are submitted that intend to remove the prefix?

Reply:
As far as I know, they are just instructional (API docs). Though, I see what you mean about removing the prefix. I was on the fence with the "default" fields, I'll get rid of them. They aren't really defaults, they are initial settings. I suspect there isn't a json-schema field for that. 😄

Reply:
> I suspect there isn't a json-schema field for that. 😄

I suspect not